lib/KBaseReport/KBaseReportImpl.pm

```diff
@@ -1,1048 +0,0 @@
-package KBaseReport::KBaseReportImpl;
```

Comment:
@scanon: no problem, your changes are covering mine completely. Let me close my PR and run tests for yours...